### ISSUE TITLE
Group Winner badge

### ISSUE THREAD

**Issue by lcpdenijs** [Slanfan/MTG-Tolaria#260]
As soon as the tournament starts the group winner badge is given to the player who is on top of the group depending on the parameters for group win.
Would it be possible to assign the group winner badge only when all the matches are played?

**Comment by Slanfan** [comment]
The Badge is awarded the top seed in each group. Of course this could be hidden until all matches have been played but it will still be the same in the end.

**Comment by lcpdenijs** [comment]
Yes I know, it's not that big of an issue but it just looks kinda strange to already see a Group Winner badge when there is just 1 match played so far and would just make more sense when it would be awarded when the group is done
